src/components/Comments/Comments.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { useEffect, useState } from 'react';
+import './Comments.css';
+import { fetchCommentsByArticleId } from '../../utils/utils.js';
+import Comment from '../Comment/Comment.jsx';
+
+const Comments = ({ articleId }) => {
+
+  const [isLoading, setIsLoading] = useState(true);
+  const [comments, setComments] = useState([]);
+
+  useEffect(() => {
+    fetchCommentsByArticleId(articleId).then((items) => {
+      setComments(items);
+      setIsLoading(false);
+    });
+  }, []);
```

Comment:
What if there's an error fetching the comments? Consider some error handling and an `isError` pattern.